source/_components/climate.openenergymonitor.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: "Open Energymonitor WiFi Thermostat"
+description: "Instructions how to integrate the Open Energy Monitor thermostat with Home Assistant."
+date: 2017-01-27 11:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Climate
+---
+
+This platform supports the ESP82666 based "WiFi MQTT Relay / Thermostat" sold
+by
+[OpenEnergyMonitor](https://shop.openenergymonitor.com/wifi-mqtt-relay-thermostat/).
+The underlying [library](http://oemthermostat.readthedocs.io/) only supports
+this single relay variant of the [original
+device](https://harizanov.com/2014/12/wifi-iot-3-channel-relay-board-with-mqtt-and-http-api-using-esp8266/).
+
+This platform controls the setpoint of the thermostat in it's "manual" mode,
+therefore there are configuration options for the initial setpoint and the away
+setpoint.
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+climate oem:
+  - platform: oem
+    name: Home
+    host: 192.168.0.100
+    port: 80
+    target_temp: 19
+    away_temp: 14
+```
```

Comment:
Please keep this sample minimized. This avoid issues caused by copy-&-paste for users. If needed, add the full example at the end of the page.